core/deployment/src/main/java/io/quarkus/deployment/index/IndexingUtil.java

```diff
@@ -273,10 +273,10 @@ public void visitPath(PathVisit visit) {
     }
 
     private static class MetaInfJandexReader implements Function<PathVisit, Index> {
-        private static MetaInfJandexReader instance;
+        private final String treeDescription;
```

Comment:
I added this because `visit.getRoot()` often returns just `/` (e.g. in case of a JAR from local Maven repo), so that's quite useless. `PathTree.toString()` is much more descriptive (e.g. in case of a JAR from local Maven repo, it returns the full path).